common/djangoapps/student/helpers.py

```diff
@@ -230,3 +235,38 @@ def get_next_url_for_login_page(request):
         # be saved in the session as part of the pipeline state. That URL will take priority
         # over this one.
     return redirect_to
+
+
+def get_profile_visibility(user_profile, user, configuration=None):
```

Comment:
Why is this code moved here instead of exposed through something like `user_api.profiles.api.py`?
@cahrens Is there a recommended path for exposing user profile information to other django projects?